.github/workflows/automation-ondemand-tests.yml

```diff
@@ -185,7 +204,7 @@ jobs:
             echo "upgrade_version=${{ inputs.chainlinkVersion }}" >>$GITHUB_OUTPUT
             echo "upgrade_image=$READ_CL_IMAGE" >>$GITHUB_OUTPUT
           fi
-          if [[ "${{ matrix.tests.name }}" == "upgrade" ]]; then
+          if [[ "${{ matrix.tests.type }}" == "upgrade" ]]; then
```

Comment:
is this unrelated change?

Reply:
this is a required change to fix the upgrade test.